core/field_label.js

```diff
@@ -88,6 +96,14 @@ Blockly.FieldLabel.fromJson = function(options) {
  */
 Blockly.FieldLabel.prototype.EDITABLE = false;
 
+/**
+ * @override
+ */
+Blockly.FieldLabel.prototype.configure_ = function(config) {
+  Blockly.FieldLabel.superClass_.configure_.call(this, config);
+  this.class_ = config['class'] || null;
```

Comment:
Theoretically you can set a class name to `0`?

Reply:
Theoretically, yes.

`utils.dom.addClass` can handle that through implicit casts (tested), but we can caste explicitly or throw errors if you want.

Reply:
Yes, so I think we just remove the null fallback here, because passing config['class'] currently would set ``this.class_`` to null.

Reply:
Ohhh sorry I thought you were pointing out something about typing, fixed.